### DISCUSSION TITLE
Naming the bluetooth network

### DISCUSSION THREAD

**Discussion by pkerga** [darthcloud/BlueRetro#274] [Ideas]
Is it possible to consider letting the user choose the name of the bluetooth network?  For example from "Blueretro" to "Blueretro PSX P1"?
Thanks !

**Comment by darthcloud** [comment]
Not user set but next version will generate a unique name for each ESP in the style of BlueRetro-PSX-54D7 (end is last 2 digit of MAC).

**Comment by pkerga** [comment]
That's a great news and a great idear !
